vignettes/taudem-installation.Rmd

```diff
@@ -29,9 +29,6 @@ You're in luck!
 The TauDEM Windows installer will install all three tools at once! 
 Refer to <https://hydrology.usu.edu/taudem/taudem5/downloads.html>.
 
-```{r, child="../man/fragments/_sitrep.Rmd"}
```

Comment:
why remove this fragment though?

Reply:
Because it refers to all 3 OSs, while the instructions above are only for Windows. Instructions for macOS and Linux only come later on. Or am I missing something?

Reply:
Fair enough! I put it three times to be sure that someone only reading the Windows section would still see it, for instance.

Reply:
To me it looks a bit confusing if we repeat the same thing three times. Maybe we could add an initial sentence like 'follow instructions on how to install TauDEM depending on your OS, then finalize your installation as specified below'?

Reply:
Sounds good to me if you add this sentence! :heavy_check_mark: